coops-ui/src/components/Common/OfficeAddresses.vue

```diff
@@ -57,13 +57,11 @@
       </li>
 
       <!-- Registered Mailing Address -->
-      <li class="container">
+      <li class="address-list-container">
         <div class="meta-container">
           <label v-if="showAddressForm">Mailing Address</label>
-          <label v-else></label>
```

Comment:
Have you tested how the columns arrange without this?

Reply:
I did not think of testing this, so I will do it now.

Reply:
Good catch. Yes, this broke the column alignment, so I'm restoring the code (but in a way that makes it intentional).